.eslintrc.json

```diff
@@ -20,7 +20,6 @@
     "comma-dangle": ["warn", "only-multiline"],
     "quotes": [
       "error",
-      "single",
```

Comment:
Removing this line causes the page to not load correctly. Be careful when making changes to files like this.